keyboards/vitamins_included/rev1/rules.mk

```diff
@@ -1,5 +1,8 @@
-BACKLIGHT_ENABLE = no
-AUDIO_ENABLE = yes
-RGBLIGHT_ENABLE = yes
-DEBUG_ENABLE = no
-CONSOLE_ENABLE = no
+SRC += matrix.c \
```

Comment:
Just a heads up, any hardware interfacing code should really be added to `QUANTUM_LIB_SRC` rather than `SRC`.  That prevents it from being optimized, if LTO is enabled, which can cause issues. 

Specifically, i2c and serial fall under this. 

Additionally, ideally, we should use `i2c_master.c` here, rather than a custom implementation of i2c. 

I know this block is just being moved .... but.... at least for the future, should be switched to i2c_master.c.

Reply:
Do you want this to be done in this PR or a separate one?

Reply:
I would prefer now, since it affects the stability of the boards.

Reply:
right, sorry I wasn't too clear. I meant the switch to i2c_master :)

Reply:
Ideally, yes, that as well.  The fewer custom implementations of i2c that we have, the better. :)

Reply:
I'll give it a look. A quick look over has them looking a bit different.

Reply:
@drashna So I looked it over, and the custom i2c implementation is a good bit different. I can port rev1 to use the generic driver, but I'd rather not invest the time, as I plan to move rev1 to split common entirely once/if this PR gets merged: https://github.com/qmk/qmk_firmware/pull/6424